.openshift/cron/hourly/git_repos

```diff
@@ -42,6 +42,8 @@ update_repo foreman_snapshot https://github.com/theforeman/foreman_snapshot
 update_repo foreman-tasks https://github.com/theforeman/foreman-tasks
 update_repo foreman-xen https://github.com/theforeman/foreman-xen
 update_repo hammer-cli-csv https://github.com/Katello/hammer-cli-csv.git
+update_repo hammer_cli_foreman_bootdisk https://github.com/theforeman/hammer_cli_foreman_bootdisk.git
+update_repo hammer-cli-foreman-discovery https://github.com/theforeman/hammer-cli-foreman-discovery
```

Comment:
@domcleal I don't see it in the master branch.
https://github.com/theforeman/redmine/blob/master/.openshift/cron/hourly/git_repos#L47